sbt-dotty/sbt-test/sbt-dotty/compiler-plugin/test

```diff
@@ -1,5 +1,4 @@
 > plugin/publishLocal
+> clean
+> compile
```

Comment:
I have been playing around with this and the plugin doesn't seems to be applied. I changed the plugin to emit errors instead of warnings. Then this compile should fail but it does not

Reply:
Yes, exactly, thanks for looking into it. I was debugging the whole day,  sbt somehow ignored the setting. If I pass `-Xplugin` manually, it works without problem.